Python/bltinmodule.c

```diff
@@ -243,38 +243,41 @@ PyDoc_STRVAR(build_class_doc,
 \n\
 Internal helper function used by the class statement.");
 
+/*[clinic input]
+__import__ as builtin___import__
+
+    name: object
```

Comment:
```suggestion
    name: unicode
```

Reply:
It is intentional to avoid checking twice if it is str.
See https://github.com/python/cpython/blob/d1777515f9f53b452a4231d68196a7c0e5deb879/Python/import.c#L1789